mne/report.py

```diff
@@ -772,7 +774,7 @@ def add_section(self, figs, captions, section='custom'):
                                          section=section)
 
     def add_figs_to_section(self, figs, captions, section='custom',
-                            scale=None, image_format='png'):
+                            scale=None, image_format='png', textbox=None):
```

Comment:
is it actually a box? can't think of a better name ... a caption maybe? but we already have that